websites/R/REYFM/presence.ts

```diff
@@ -183,6 +196,8 @@ presence.on("UpdateData", async () => {
       presenceData.state = format2
         .replace("%title%", track)
         .replace("%artist%", artist);
+
+      presenceData.buttons = [{ label: "Listen along!", url: document.URL }];
```

Comment:
You sure it will not redirect to main page in any way?

Reply:
Fixed.